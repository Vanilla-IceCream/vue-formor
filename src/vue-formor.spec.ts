import { describe, it, expect } from 'vitest';
import { mount } from '@vue/test-utils';

import BasicForm from './fixtures/BasicForm.vue';

describe('vue-formor', () => {
  it('BasicForm', () => {
    const wrapper = mount(BasicForm, {});

    expect(wrapper.html()).toMatchSnapshot();
  });
});
