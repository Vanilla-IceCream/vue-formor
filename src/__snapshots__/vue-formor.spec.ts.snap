// Vitest Snapshot v1

exports[`vue-formor > BasicForm 1`] = `
"<div>
  <div>Basic Forms</div>
  <div>
    <div><label for=\\"email\\">Email:</label><input id=\\"email\\" type=\\"email\\">
      <div></div>
    </div>
    <div><label for=\\"password\\">Password:</label><input id=\\"password\\" type=\\"password\\">
      <div></div>
    </div><button>Sign in</button>
  </div><pre>{}</pre><pre>{}</pre>
</div>"
`;
